nvflare/job_config/stats_job.py

```diff
@@ -37,7 +37,7 @@ def __init__(
         max_noise_level=0.3,
         max_bins_percent=10,
     ):
-        super().__init__()
+        super().__init__(name=job_name)
         self.writer_id = "stats_writer"
         self.stats_generator_id_prefix = "stats_generator"
         self.job_name = job_name
```

Comment:
should we get rid of the "job_name" argument and just use self.name in the parent class?